process/sync/baseSync.go

```diff
@@ -502,6 +502,11 @@ func (boot *baseBootstrap) syncBlocks(ctx context.Context) {
 
 		err := boot.syncStarter.SyncBlock()
```

Comment:
There is another err log in this method which could be avoided if we also send ctx there. I don't have a strong opinion if it is mandatory to be done also there, as we log here that the go routine is stopping.

Reply:
Right, the correct process there is to replace the error